static/js/search.js

```diff
@@ -184,6 +184,7 @@ function doSearch(data) {
                 reject(errorMessages);
             }
             console.timeEnd(timerName);
+            initializeFilterInputEvents();
```

Comment:
No need to call `initializeFilterInputEvents` here, as we already have the call in `log-search.js` on document load.